src/Tasks/Microsoft.NET.Build.Tasks/ConflictResolution/NuGetUtilities.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using System;
+using System.IO;
+
+namespace Microsoft.NET.Build.Tasks.ConflictResolution
+{
+    static partial class NuGetUtilities
```

Comment:
Can this just go in [https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/NuGetUtils.cs](https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/NuGetUtils.cs) so we don't have 2?